packages/api/index.ts

```diff
@@ -1,16 +1,41 @@
-import ExpressApolloApp from './server'
+import ExpressServer, { Express } from 'express'
+import { ApolloServer, ApolloServerExpressConfig } from 'apollo-server-express'
+import GraphQLServer from './apollo-server'
+import Mongoose from 'mongoose'
 import config from './config'
 
-const { DOMAIN, PORT, GRAPHQL_ENDPOINT } = config.API
-const SERVER_DOMAIN = config.ENV === 'production' ? DOMAIN : `${DOMAIN}:${PORT}`
-
-const main = async () => {
-  const app = await ExpressApolloApp({
-    graphqlEndpoint: GRAPHQL_ENDPOINT
+const startMongoConnection = async () => {
+  return Mongoose.connect('mongodb://localhost/test', {
```

Comment:
Is `test` too generic? I have this test db as a playground. Prefer to have `codereviewer` or `crdb` as database name.

Reply:
Think this is already in the `master` branch, but yes, we should use an environment variable which is already setup in the `config.ts`. I'll fix it tonight.